merlin/systems/dag/ops/pytorch.py

```diff
@@ -93,6 +96,17 @@ def compute_output_schema(
         """
         return self.output_schema
 
+    def transform(self, col_selector: ColumnSelector, transformable: Transformable):
+        tensor_dict = {}
+        for column in transformable.columns:
+            tensor_dict[column] = torch.from_numpy(np.squeeze(transformable[column].values))
+
+        result = self.model(tensor_dict)
+        output = {}
+        for idx, col in enumerate(self.output_schema.column_names):
```

Comment:
This allows for more than one output, but ordering will matter at this point. Is there a better way to go about handling a multi output where we can disregard order? Luckily dictionaries hold on to order...

Reply:
In this case, it seems like letting the model return a dictionary is okay, since we don't have the limitations that the Triton Torch backend imposes on us 🤔